ui/components/MesheryApplications.js

```diff
@@ -125,6 +126,21 @@ function MesheryApplications({ updateProgress, enqueueSnackbar, closeSnackbar, u
    * @param {string} search search string
    * @param {string} sortOrder order of sort
    */
+
+  const handleDeploy = (application_file) => {
```

Comment:
add `updateProgress({ showProgress: true })` inside this function so that user can get the feedback that his action has been registered.